aarch64/linux fails to compile

it looks like the whole assembly section is missing when compiling with linux under arm:

```
[ 97%] Linking CXX executable mksnapshot
/usr/bin/ld: CMakeFiles/mksnapshot.dir/v8/src/snapshot/mksnapshot.cc.o: in function `v8::internal::CpuFeatures::Probe(bool)':
mksnapshot.cc:(.text._ZN2v88internal11CpuFeatures5ProbeEb[_ZN2v88internal11CpuFeatures5ProbeEb]+0x34): undefined reference to `v8::internal::CpuFeatures::ProbeImpl(bool)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::(anonymous namespace)::BuildPlaceholder(v8::internal::Isolate*, v8::internal::Builtin)':
setup-builtins-internal.cc:(.text+0x1d8): undefined reference to `v8::internal::MacroAssembler::Move(v8::internal::Register, v8::internal::Smi)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f0): undefined reference to `v8::internal::MacroAssembler::Call(v8::internal::Register)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::(anonymous namespace)::BuildWithMacroAssembler(v8::internal::Isolate*, v8::internal::Builtin, void (*)(v8::internal::MacroAssembler*), char const*)':
setup-builtins-internal.cc:(.text+0x3f0): undefined reference to `v8::internal::Assembler::GetCode(v8::internal::Isolate*, v8::internal::CodeDesc*, v8::internal::SafepointTableBuilderBase*, int)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::(anonymous namespace)::BuildAdaptor(v8::internal::Isolate*, v8::internal::Builtin, unsigned long, char const*)':
setup-builtins-internal.cc:(.text+0x530): undefined reference to `v8::internal::Builtins::Generate_Adaptor(v8::internal::MacroAssembler*, unsigned long)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::SetupIsolateDelegate::SetupBuiltinsInternal(v8::internal::Isolate*)':
setup-builtins-internal.cc:(.text+0xb74): undefined reference to `v8::internal::Builtins::Generate_DeoptimizationEntry_Eager(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xb78): undefined reference to `v8::internal::Builtins::Generate_DeoptimizationEntry_Eager(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xbb0): undefined reference to `v8::internal::Builtins::Generate_DeoptimizationEntry_Lazy(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xbb4): undefined reference to `v8::internal::Builtins::Generate_DeoptimizationEntry_Lazy(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1444): undefined reference to `v8::internal::Builtins::Generate_ConstructFunction(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1448): undefined reference to `v8::internal::Builtins::Generate_ConstructFunction(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1480): undefined reference to `v8::internal::Builtins::Generate_ConstructBoundFunction(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1484): undefined reference to `v8::internal::Builtins::Generate_ConstructBoundFunction(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x14bc): undefined reference to `v8::internal::Builtins::Generate_ConstructedNonConstructable(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x14c0): undefined reference to `v8::internal::Builtins::Generate_ConstructedNonConstructable(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x14f8): undefined reference to `v8::internal::Builtins::Generate_Construct(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x14fc): undefined reference to `v8::internal::Builtins::Generate_Construct(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x17a8): undefined reference to `v8::internal::Builtins::Generate_JSConstructStubGeneric(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x17ac): undefined reference to `v8::internal::Builtins::Generate_JSConstructStubGeneric(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x17e4): undefined reference to `v8::internal::Builtins::Generate_JSBuiltinsConstructStub(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x17e8): undefined reference to `v8::internal::Builtins::Generate_JSBuiltinsConstructStub(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x18e0): undefined reference to `v8::internal::Builtins::Generate_JSEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x18e4): undefined reference to `v8::internal::Builtins::Generate_JSEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x191c): undefined reference to `v8::internal::Builtins::Generate_JSConstructEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1920): undefined reference to `v8::internal::Builtins::Generate_JSConstructEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1958): undefined reference to `v8::internal::Builtins::Generate_JSRunMicrotasksEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x195c): undefined reference to `v8::internal::Builtins::Generate_JSRunMicrotasksEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1994): undefined reference to `v8::internal::Builtins::Generate_JSEntryTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1998): undefined reference to `v8::internal::Builtins::Generate_JSEntryTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x19d0): undefined reference to `v8::internal::Builtins::Generate_JSConstructEntryTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x19d4): undefined reference to `v8::internal::Builtins::Generate_JSConstructEntryTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1a0c): undefined reference to `v8::internal::Builtins::Generate_ResumeGeneratorTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1a10): undefined reference to `v8::internal::Builtins::Generate_ResumeGeneratorTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1e68): undefined reference to `v8::internal::Builtins::Generate_InterpreterEnterAtBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1e6c): undefined reference to `v8::internal::Builtins::Generate_InterpreterEnterAtBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1ea4): undefined reference to `v8::internal::Builtins::Generate_InterpreterEnterAtNextBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1ea8): undefined reference to `v8::internal::Builtins::Generate_InterpreterEnterAtNextBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1ee0): undefined reference to `v8::internal::Builtins::Generate_InterpreterOnStackReplacement(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1ee4): undefined reference to `v8::internal::Builtins::Generate_InterpreterOnStackReplacement(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f1c): undefined reference to `v8::internal::Builtins::Generate_BaselineOutOfLinePrologue(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f20): undefined reference to `v8::internal::Builtins::Generate_BaselineOutOfLinePrologue(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f58): undefined reference to `v8::internal::Builtins::Generate_BaselineOutOfLinePrologueDeopt(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f5c): undefined reference to `v8::internal::Builtins::Generate_BaselineOutOfLinePrologueDeopt(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f94): undefined reference to `v8::internal::Builtins::Generate_BaselineOnStackReplacement(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x1f98): undefined reference to `v8::internal::Builtins::Generate_BaselineOnStackReplacement(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x200c): undefined reference to `v8::internal::Builtins::Generate_BaselineOrInterpreterEnterAtBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2010): undefined reference to `v8::internal::Builtins::Generate_BaselineOrInterpreterEnterAtBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2048): undefined reference to `v8::internal::Builtins::Generate_BaselineOrInterpreterEnterAtNextBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x204c): undefined reference to `v8::internal::Builtins::Generate_BaselineOrInterpreterEnterAtNextBytecode(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2084): undefined reference to `v8::internal::Builtins::Generate_InterpreterOnStackReplacement_ToBaseline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2088): undefined reference to `v8::internal::Builtins::Generate_InterpreterOnStackReplacement_ToBaseline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x21bc): undefined reference to `v8::internal::Builtins::Generate_NotifyDeoptimized(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x21c0): undefined reference to `v8::internal::Builtins::Generate_NotifyDeoptimized(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x21f8): undefined reference to `v8::internal::Builtins::Generate_ContinueToCodeStubBuiltin(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x21fc): undefined reference to `v8::internal::Builtins::Generate_ContinueToCodeStubBuiltin(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2234): undefined reference to `v8::internal::Builtins::Generate_ContinueToCodeStubBuiltinWithResult(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2238): undefined reference to `v8::internal::Builtins::Generate_ContinueToCodeStubBuiltinWithResult(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2270): undefined reference to `v8::internal::Builtins::Generate_ContinueToJavaScriptBuiltin(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2274): undefined reference to `v8::internal::Builtins::Generate_ContinueToJavaScriptBuiltin(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x22ac): undefined reference to `v8::internal::Builtins::Generate_ContinueToJavaScriptBuiltinWithResult(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x22b0): undefined reference to `v8::internal::Builtins::Generate_ContinueToJavaScriptBuiltinWithResult(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x22e8): undefined reference to `v8::internal::Builtins::Generate_CallApiCallback(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x22ec): undefined reference to `v8::internal::Builtins::Generate_CallApiCallback(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2324): undefined reference to `v8::internal::Builtins::Generate_CallApiGetter(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2328): undefined reference to `v8::internal::Builtins::Generate_CallApiGetter(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2660): undefined reference to `v8::internal::Builtins::Generate_RestartFrameTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x2664): undefined reference to `v8::internal::Builtins::Generate_RestartFrameTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x31dc): undefined reference to `v8::internal::Builtins::Generate_RunMicrotasksTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x31e0): undefined reference to `v8::internal::Builtins::Generate_RunMicrotasksTrampoline(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x5f98): undefined reference to `v8::internal::Builtins::Generate_FunctionPrototypeApply(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x5f9c): undefined reference to `v8::internal::Builtins::Generate_FunctionPrototypeApply(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x6014): undefined reference to `v8::internal::Builtins::Generate_FunctionPrototypeCall(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x6018): undefined reference to `v8::internal::Builtins::Generate_FunctionPrototypeCall(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x9150): undefined reference to `v8::internal::Builtins::Generate_ReflectApply(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x9154): undefined reference to `v8::internal::Builtins::Generate_ReflectApply(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x918c): undefined reference to `v8::internal::Builtins::Generate_ReflectConstruct(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0x9190): undefined reference to `v8::internal::Builtins::Generate_ReflectConstruct(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa6c0): undefined reference to `v8::internal::Builtins::Generate_GenericJSToWasmWrapper(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa6c4): undefined reference to `v8::internal::Builtins::Generate_GenericJSToWasmWrapper(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa6fc): undefined reference to `v8::internal::Builtins::Generate_WasmReturnPromiseOnSuspend(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa700): undefined reference to `v8::internal::Builtins::Generate_WasmReturnPromiseOnSuspend(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa738): undefined reference to `v8::internal::Builtins::Generate_WasmSuspend(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa73c): undefined reference to `v8::internal::Builtins::Generate_WasmSuspend(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa774): undefined reference to `v8::internal::Builtins::Generate_WasmResume(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa778): undefined reference to `v8::internal::Builtins::Generate_WasmResume(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa7b0): undefined reference to `v8::internal::Builtins::Generate_WasmReject(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa7b4): undefined reference to `v8::internal::Builtins::Generate_WasmReject(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa7ec): undefined reference to `v8::internal::Builtins::Generate_WasmCompileLazy(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa7f0): undefined reference to `v8::internal::Builtins::Generate_WasmCompileLazy(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa828): undefined reference to `v8::internal::Builtins::Generate_WasmLiftoffFrameSetup(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa82c): undefined reference to `v8::internal::Builtins::Generate_WasmLiftoffFrameSetup(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa864): undefined reference to `v8::internal::Builtins::Generate_WasmDebugBreak(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa868): undefined reference to `v8::internal::Builtins::Generate_WasmDebugBreak(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa8a0): undefined reference to `v8::internal::Builtins::Generate_WasmOnStackReplace(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xa8a4): undefined reference to `v8::internal::Builtins::Generate_WasmOnStackReplace(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xb704): undefined reference to `v8::internal::Builtins::Generate_DirectCEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xb708): undefined reference to `v8::internal::Builtins::Generate_DirectCEntry(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xb7c0): undefined reference to `v8::internal::Builtins::Generate_DoubleToI(v8::internal::MacroAssembler*)'
/usr/bin/ld: setup-builtins-internal.cc:(.text+0xb7c4): undefined reference to `v8::internal::Builtins::Generate_DoubleToI(v8::internal::MacroAssembler*)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::Assembler::GetCode(v8::internal::Isolate*, v8::internal::CodeDesc*)':
setup-builtins-internal.cc:(.text._ZN2v88internal9Assembler7GetCodeEPNS0_7IsolateEPNS0_8CodeDescE[_ZN2v88internal9Assembler7GetCodeEPNS0_7IsolateEPNS0_8CodeDescE]+0x28): undefined reference to `v8::internal::Assembler::GetCode(v8::internal::Isolate*, v8::internal::CodeDesc*, v8::internal::SafepointTableBuilderBase*, int)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::Assembler::target_pointer_address_at(unsigned long)':
setup-builtins-internal.cc:(.text._ZN2v88internal9Assembler25target_pointer_address_atEm[_ZN2v88internal9Assembler25target_pointer_address_atEm]+0x18): undefined reference to `v8::internal::Instruction::ImmPCOffsetTarget()'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::Assembler::target_address_at(unsigned long, unsigned long)':
setup-builtins-internal.cc:(.text._ZN2v88internal9Assembler17target_address_atEmm[_ZN2v88internal9Assembler17target_address_atEmm]+0x44): undefined reference to `v8::internal::Instruction::ImmPCOffsetTarget()'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::Assembler::set_target_address_at(unsigned long, unsigned long, unsigned long, v8::internal::ICacheFlushMode)':
setup-builtins-internal.cc:(.text._ZN2v88internal9Assembler21set_target_address_atEmmmNS0_15ICacheFlushModeE[_ZN2v88internal9Assembler21set_target_address_atEmmmNS0_15ICacheFlushModeE]+0x70): undefined reference to `v8::internal::Instruction::SetBranchImmTarget(v8::internal::Instruction*)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::FrameScope::FrameScope(v8::internal::MacroAssembler*, v8::internal::StackFrame::Type)':
setup-builtins-internal.cc:(.text._ZN2v88internal10FrameScopeC2EPNS0_14MacroAssemblerENS0_10StackFrame4TypeE[_ZN2v88internal10FrameScopeC5EPNS0_14MacroAssemblerENS0_10StackFrame4TypeE]+0x70): undefined reference to `v8::internal::MacroAssembler::EnterFrame(v8::internal::StackFrame::Type)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::FrameScope::~FrameScope()':
setup-builtins-internal.cc:(.text._ZN2v88internal10FrameScopeD2Ev[_ZN2v88internal10FrameScopeD5Ev]+0x44): undefined reference to `v8::internal::MacroAssembler::LeaveFrame(v8::internal::StackFrame::Type)'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::MacroAssemblerBase::~MacroAssemblerBase()':
setup-builtins-internal.cc:(.text._ZN2v88internal18MacroAssemblerBaseD2Ev[_ZN2v88internal18MacroAssemblerBaseD5Ev]+0x20): undefined reference to `v8::internal::Assembler::~Assembler()'
/usr/bin/ld: libv8_base_without_compiler.a(setup-builtins-internal.cc.o): in function `v8::internal::MacroAssembler::MacroAssemblerBase(v8::internal::Isolate*, v8::internal::CodeObjectRequired, std::unique_ptr<v8::internal::AssemblerBuffer, std::default_delete<v8::internal::AssemblerBuffer> >)':
setup-builtins-internal.cc:(.text._ZN2v88internal14MacroAssemblerCI2NS0_18MacroAssemblerBaseEEPNS0_7IsolateENS0_18CodeObjectRequiredESt10unique_ptrINS0_15AssemblerBufferESt14default_deleteIS7_EE[_ZN2v88internal14MacroAssemblerCI5NS0_18MacroAssemblerBaseEEPNS0_7IsolateENS0_18CodeObjectRequiredESt10unique_ptrINS0_15AssemblerBufferESt14default_deleteIS7_EE]+0x30): undefined reference to `vtable for v8::internal::MacroAssembler'
/usr/bin/ld: setup-builtins-internal.cc:(.text._ZN2v88internal14MacroAssemblerCI2NS0_18MacroAssemblerBaseEEPNS0_7IsolateENS0_18CodeObjectRequiredESt10unique_ptrINS0_15AssemblerBufferESt14default_deleteIS7_EE[_ZN2v88internal14MacroAssemblerCI5NS0_18MacroAssemblerBaseEEPNS0_7IsolateENS0_18CodeObjectRequiredESt10unique_ptrINS0_15AssemblerBufferESt14default_deleteIS7_EE]+0x34): undefined reference to `vtable for v8::internal::MacroAssembler'
```

(and so on for another 5000+ lines)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,7 +65,7 @@ set(is-msvc $<CXX_COMPILER_ID:MSVC>)
 # FIXME obviously
 set(is-arm 0)
 set(is-arm64 0)
-if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "arm64")
+if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "arm64" OR CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
   set(is-arm64 1)
 endif ()
 set(is-ia32 0)

